src/modules/analysis/AnalysisViewComponent.js

```diff
@@ -14,29 +14,31 @@ import {
   Panel
 } from 'react-bootstrap';
 import FontAwesome from 'react-fontawesome';
-
-type Props = { store: any };
+import {SaveCancel} from '../../components/formfields/index';
+import {hashHistory} from 'react-router';
+import Config from '../../config';
+type Props = { store: any, appSession: any, params: any };
```

Comment:
i think you should iron out the properties you are using on these three Props fields.